### ISSUE TITLE
(Don't show) How to enable "Last seen and last message" option? Can't see this information in chat page.

### ISSUE THREAD

**Issue by LXSysadmin** [RocketChat/Rocket.Chat#11284]
Hello!
How to enable feature to see user last activity time and channel or chat last message? Please see picture attached. We can't see this information and can't find any setting to enable this. We have Rocket Chat server for some 2 years using docker technologies. 

![chat](https://user-images.githubusercontent.com/19779209/42022030-d30a36ae-7ac4-11e8-9223-1662e3ae715f.jpg)

### Server Setup Information:
- Operating System: Ubuntu 16
- Deployment Method: Docker
- Number of Running Instances: 1
- MongoDB Version: Default by docker deployment 

Maybe we have some old database version or this depends that we are using dockers?

**Comment by teamsoo** [comment]
What RC version are you running?
  The setting is in `Administration -> General`
<img width="1102" alt="screen shot 2561-06-28 at 15 28 47" src="https://user-images.githubusercontent.com/9608690/42022661-0e73ab94-7ae8-11e8-9518-6feaa9ad77e3.png">

**Comment by LXSysadmin** [comment]
Sorry, RC version 0.66.0
Yes, i found it. Also view mode must be set to Extended. If you have mode Mediums or Condensed you can't see Last message. This is good. But where to enable last seen time? Now, we can see last message but is there setting to enable "user last seen"? Or this date show last message date? :)

**Comment by TwizzyDizzy** [comment]
Nope, the last message date is the date of the last message. I'm pretty sure that there is not yet a feature that implements a "last seen" on a contact basis. I'm heavily opposed to such stuff, yet I see how this could be a legitimate requirement. Pushing this into the review loop.

Cheers
Thomas

**Comment by vynmera** [comment]
I feel this would be a _very_ worthy addition to workspace environments (or even public environments). Slack has it too ;)

**Comment by partosXP** [comment]
I really can't understand, how r you all using chat/messenger whithout this feature.
Almost all  of them have it: Skype for business, Telegram, WhatsApp etc. Whant it here too...
The same closed issue: #823

**Comment by arpaplus** [comment]
Any news? we need to know logoff time as well!

**Comment by andanotherusername** [comment]
This is a really important feature. Would love to see this implemented.

**Comment by caesarcxiv** [comment]
+1

**Comment by ali-babaei** [comment]
+1
